src/core/cita_bft.rs

```diff
@@ -50,24 +50,40 @@ const INIT_ROUND: usize = 0;
 
 const MAX_PROPOSAL_TIME_COEF: usize = 10;
 
-const LOG_TYPE_PROPOSE: u8 = 1;
-const LOG_TYPE_VOTE: u8 = 2;
-const LOG_TYPE_STATE: u8 = 3;
-const LOG_TYPE_PREV_HASH: u8 = 4;
-const LOG_TYPE_COMMITS: u8 = 5;
-const LOG_TYPE_VERIFIED_PROPOSE: u8 = 6;
-const LOG_TYPE_AUTH_TXS: u8 = 7;
-
 const TIMEOUT_RETRANSE_MULTIPLE: u32 = 15;
 const TIMEOUT_LOW_ROUND_MESSAGE_MULTIPLE: u32 = 20;
 
-const VERIFIED_PROPOSAL_OK: i8 = 1;
-const VERIFIED_PROPOSAL_FAILED: i8 = -1;
-const VERIFIED_PROPOSAL_UNDO: i8 = 0;
-
 pub type TransType = (String, Vec<u8>);
 pub type PubType = (String, Vec<u8>);
 
+#[derive(Debug, PartialEq, PartialOrd, Clone, Copy)]
+enum VerifiedProposalStatus {
+    Ok,
+    Failed,
+    Undo,
+}
```

Comment:
enum Foo {
    A = 1,
   B = -1,
   C = 0, 
}

Reply:
Default is not `u8`.

Reply:
https://github.com/cryptape/cita-bft/blob/develop/src/core/wal.rs#L140

Reply:
There are two enum, `u8` is another one, this is `i8`.